src/instructlab/model/download.py

```diff
@@ -337,11 +343,12 @@ def download(ctx, repositories, releases, filenames, model_dir, hf_token):
 
         try:
             downloader.download()
+            click.echo(f"\n✅ {repository} model download completed successfully!\n")
```

Comment:
might want to use `downloader.repository` here for consistency

Reply:
thanks for your remind.. updated. @jaideepr97